cli/config/configuration-fc.toml.in

```diff
@@ -16,7 +16,7 @@ path = "@FCPATH@"
 # If the jailer path is not set kata will launch firecracker
 # without a jail. If the jailer is set firecracker will be
 # launched in a jailed enviornment created by the jailer
-jailer_path = "@FCJAILERPATH@"
+#jailer_path = "@FCJAILERPATH@"
```

Comment:
can we update the comment to note why it is disabled by default?

Reply:
@grahamwhaley I have added a comment. @egernst Let me know if that looks ok to you.